src/index_improved.ts

```diff
@@ -36,6 +36,6 @@
 
 import ARControllerNFT from "./ARControllerNFT_improved";
 
-export default {
```

Comment:
this fix the issue #292 i have tested with ARnft

Reply:
Now you'll be able to import the ARControllerNFT with:

`import { ARControllerNFT } from '@webarkit/jsartoolkit-nft'`